sklearn/cluster/spectral.py

```diff
@@ -210,10 +248,10 @@ def spectral_clustering(affinity, n_clusters=8, n_components=None,
         Stopping criterion for eigendecomposition of the Laplacian matrix
         when using arpack eigen_solver.
 
-    assign_labels : {'kmeans', 'discretize'}, default: 'kmeans'
+    assign_labels : {'kmeans', 'discretize', 'clusterQR'}, default: 'kmeans'
```

Comment:
should this just be `qr` rather than `clusterQR`? I think we should avoid caps, and prefer `cluster-qr`.

Reply:
The original authors call it  `clusterQR` in their paper. It is surely not just `qr`.

I have no personal opinion about the name to use here, and would be happy to rename all instances of  `clusterQR`  into  `cluster-qr` if this is the preferred naming style. Please reconfirm before I make the global change.

Reply:
@jnothman  - Could you please reconfirm your request before I make the global change?